src/ORM/Marshaller.php

```diff
@@ -105,6 +105,18 @@ public function one(array $data, array $options = [])
         $propertyMap = $this->_buildPropertyMap($options);
 
         $schema = $this->_table->schema();
+        $primaryKey = $schema->primaryKey();
+
+        if (array_intersect($primaryKey, array_keys($data)) == $primaryKey) {
+            $record = $this->_table->find('all');
+            foreach ($primaryKey as $pkey) {
+                $record->where(["$pkey" => $data[$pkey] ]);
```

Comment:
Alright, them. If that's the case, I will add the table names. Thanks for your help.